django_safari_notifications/management/commands/create_icons.py

```diff
@@ -0,0 +1,84 @@
+from django.core.management.base import BaseCommand
+from django.apps import apps
+import os, sys
+from django_safari_notifications.views import ICON_SIZES
+
+config = apps.get_app_config('django_safari_notifications')
+# connects to logger in settings.py of connected project
```

Comment:
Comment is wrong. It connects to logger in config. The fact that your own logger was defined in settings.py isn't relevant to others